src/main/java/com/minecolonies/coremod/colony/permissions/Permissions.java

```diff
@@ -271,9 +272,28 @@ public void loadPermissions(@NotNull final NBTTagCompound compound)
     }
 
     /**
+     * Change the owner of a colony.
+     * @param playerName the player to set.
+     * @param world the world it comes from.
+     * @return true if succesful.
+     */
+    public boolean setOwner(final EntityPlayer player, final World world)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "world". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1172)